integration-cli/docker_cli_build_test.go

```diff
@@ -3989,7 +3988,7 @@ ADD test.tar /unlikely-to-exist
 RUN cat /unlikely-to-exist/test/foo | grep Hi
 ADD test.tar /unlikely-to-exist-trailing-slash/
 RUN cat /unlikely-to-exist-trailing-slash/test/foo | grep Hi
-RUN mkdir /existing-directory
+RUN sh -c "mkdir /existing-directory"
```

Comment:
Windows cmd also has an mkdir command (built in, rather than an executable) which takes priority over the Busybox version. It expects the folders to be in the C:\\ syntax instead of the / syntax. Using sh forces the correct version to be used.

I think the long term fix for this is to have the default shell be configurable for build commands.